### ISSUE TITLE
Question about MSE in Figure 6

### ISSUE THREAD

**Issue by JinseongP** [Y-debug-sys/Diffusion-TS#21]
Hi, I have a question about measuring MSE in Figure 6. I have adjusted the code for measuring MSE as shown below.
![image](https://github.com/Y-debug-sys/Diffusion-TS/assets/53927724/4528feb5-ef4f-41cf-b23d-c6c3734fc399)

However, I obtained different results.
The first result uses data rescaled to the original scale, while the other is a scaled version ranging from 0 to 1.

![image](https://github.com/Y-debug-sys/Diffusion-TS/assets/53927724/7807b53f-d546-4cd9-8334-2c7badfaa9d8)

1) I am wondering if the results in Figure 6 are actually RMSE of the data scaled from 0 to 1. I obtained a result of 0.02698616289235689, which closely matches the results in Figure 6. However, for the Energy dataset, I obtained 0.06649003748409096. Could you clarify if any hyperparameters were changed in the energy setting?

2) If the figure indeed shows RMSE, why didn't you use the inverse_transform method as mentioned in the MuJoCo.ipynb file?

Please correct me if I have misunderstood anything!

**Comment by Y-debug-sys** [comment]
HI,

1. Yes, you're right, because all metrics of the initial version of the paper were evaluated using the 0-1 version. In fact, I used the first 90% for training, leaving the rest for the test set, and repeated serveral time to get the average results. I didn't save the code for this part of the experiment, but the config is unchanged; perhaps you can add some dropout to prevent overfitting.
2. This is because baselines like SSSD were evaluated in this manner (on MuJoCo and Solar Dataset), I directly borrowed their results.

Thanks.
